testrunner/buckets.tf

```diff
@@ -9,10 +9,11 @@ resource "google_storage_bucket" "testrunner-results-bucket" {
   uniform_bucket_level_access = true
 }
 
-resource "google_storage_bucket_iam_member" "testrunner-results-bucket-admin" {
+resource "google_storage_bucket_iam_member" "testrunner_streamer_sa_storage_bucket_iam_role" {
+  count = "${length(var.testrunner_streamer_sa_storage_bucket_iam_roles)}"
```

Comment:
I could be wrong here but I think in more recent versions of tf you actually don't need to wrap these function invocations in `"${}"` as long as you aren't interpolating the result of the function into a larger string

Just reads a little cleaner imo.

Reply:
Thanks; that is very helpful. I noticed the two forms, but didn't realize this was a preferred change. It's definitely cleaner.